packages/grafana-ui/src/index.ts

```diff
@@ -12,3 +12,4 @@ export * from './slate-plugins';
 
 // Exposes standard editors for registries of optionsUi config and panel options UI
 export { getStandardFieldConfigs, getStandardOptionEditors } from './utils/standardEditors';
+export { Route, useLocation, useParams } from 'react-router-dom';
```

Comment:
Why export from here rather than just have people import directly?  Is this to potentially help backwards compatibility?  (if  so, can you add a comment?)

Reply:
no exports anymore, we are exposing RR to plugins and updating the plugin starters now